src/Locale/en_US/default.po

```diff
@@ -191,6 +191,9 @@ msgstr ""
 msgid "Export Filtered"
 msgstr ""
 
+msgid "Format not allowed"
+msgstr "Format choosen is not available"
```

Comment:
We usually don't _translate_ messages in this locale

use `"Format choosen is not available"` directly as message string instead of `"Format not allowed"` and update references